Tests/Microsoft.AppCenter.Crashes.Test.Windows/Utils/ErrorLogHelperTest.cs

```diff
@@ -140,6 +140,37 @@ public void GetSingleErrorLogFileDoesNotThrow(Type exceptionType)
             Assert.IsNull(errorLogFile);
         }
 
+        [TestMethod]
+        public void getErrorStorageDirectoryCreate()
+        {
+            // Mock where directory doesn't exist.
+            var mockDirectory = Mock.Of<Directory>();
+            ErrorLogHelper.Instance._crashesDirectory = mockDirectory;
+            Mock.Get(mockDirectory).Setup(d => d.Create());
+            Mock.Get(mockDirectory).Setup(d => d.Exists()).Returns(false);
+
+            var errorStorageDirectory = ErrorLogHelper.GetErrorStorageDirectory();
+
+            // Verify _crashesDirectory was created
+            Assert.IsInstanceOfType(errorStorageDirectory, typeof(Directory));
+            Mock.Get(mockDirectory).Verify(d => d.Create());
+        }
+
+        [TestMethod]
+        public void getErrorStorageDirectoryExists()
+        {
+            // Mock where directory doesn't exist.
+            var mockDirectory = Mock.Of<Directory>();
+            ErrorLogHelper.Instance._crashesDirectory = mockDirectory;
+            Mock.Get(mockDirectory).Setup(d => d.Create()).Throws(new System.Exception("Should not be called"));
```

Comment:
Won't this crash the test?  Where does the exception get caught?  I don't see it in the modified code

Reply:
@jwallra The idea is it won't be called if it already exists. Since `Create` is already checking, I'll be removing this test anyway I think.